airflow/www/utils.py

```diff
@@ -437,6 +439,42 @@ def is_utcdatetime(self, col_name):
                 isinstance(obj.impl, UtcDateTime)
         return False
 
+    # This is a local fix until https://github.com/dpgaspar/Flask-AppBuilder/pull/1493 is merged and released.
+    def get(
+        self,
+        id,
+        filters: Optional[Filters] = None,
+        select_columns: Optional[List[str]] = None,
+    ) -> Any:
+        """
+        Returns the result for a model get, applies filters and supports dotted
+        notation for joins and granular selecting query columns.
+        :param id: The model id (pk).
```

Comment:
```suggestion
        notation for joins and granular selecting query columns.

        :param id: The model id (pk).
```

Reply:
docstyle _might_ complain

Reply:
Probably - I just wholesale copied it from FAB